### PR TITLE
PLATFORM-3619 | Call cookie-syncer service to sync tracking-opt-in state between domains

### PR DESCRIPTION
**Pull request by jmail**
https://wikia-inc.atlassian.net/browse/PLATFORM-3619
https://github.com/Wikia/pandora/pull/5115

@Wikia/core-platform-team

### CODE REVIEW

File: extensions/wikia/CookieSyncer/CookieSyncer.setup.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+$wgExtensionCredits['other'][] = [
+	'name' => 'CookieSyncer Extension',
+	'author' => [
+		'Platform Team @Wikia',
+	],
+	'descriptionmsg' => 'cookiesyncer-tag-desc',
```
Comment (themech) [inline extensions/wikia/CookieSyncer/CookieSyncer.setup.php]: Isn't there supposed to be a i18n file with this message?

File: extensions/wikia/CookieSyncer/styles/CookieSyncer.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.cookie-syncer-module-iframe {
+	width: 0px;
+	height: 0px;
+	border: none;
```
Comment (themech) [inline extensions/wikia/CookieSyncer/styles/CookieSyncer.scss]: nit: sort these rules

**Comment by rogatty** [comment]
Don't merge before we make sure the sync is done only once.

File: extensions/wikia/CookieSyncer/CookieSyncer.setup.php (hunk 1 above)
Comment (rogatty) [reply extensions/wikia/CookieSyncer/CookieSyncer.setup.php]: Thankfully we don't need a message which carries creating a new Crowdin project. Instead, I used a plain `description`.

**Review by themech — APPROVED** [review]

**Comment by rogatty** [comment]
Closed in favour of https://github.com/Wikia/app/pull/16063

**Pull request closed without merging**